uint8 roll_command
uint8 pitch_command
uint8 yaw_command
